src/actions/lab.js

```diff
@@ -29,9 +34,45 @@ const setShowThermometerLabels = (showThermometerLabels) => (dispatch) =>
     payload: showThermometerLabels,
   });
 
+const setShowGrid = (showGrid) => (dispatch) =>
+  dispatch({
+    type: SET_SHOW_GRID,
+    payload: showGrid,
+  });
+
+const setShowEmittedLines = (showThermometerLabels) => (dispatch) =>
+  dispatch({
+    type: SET_SHOW_EMITTED_LINES,
+    payload: showThermometerLabels,
+  });
+
+const toggleElectrons = (payload) => (dispatch) => {
+  dispatch({
+    type: TOGGLE_ELECTRONS,
+    payload,
+  });
+};
+
+const toggleWavelengthDistribution = (payload) => (dispatch) => {
+  dispatch({
+    type: TOGGLE_SPECTRUM_BAR,
```

Comment:
Rename this to `TOGGLE_WAVELENGTH_DISTRIBUTION` here and elsewhere - makes it easier to maintain later on if someone does't know that the spectrum_bar is the same thing as the wavelength_distribution